docs/topics/Gateway.md

```diff
@@ -648,7 +648,7 @@ Sent in response to [Guild Request Members](#DOCS_TOPICS_GATEWAY/request-guild-m
 | Field    | Type                                                                | Description          |
 | -------- | ------------------------------------------------------------------- | -------------------- |
 | guild_id | snowflake                                                           | the id of the guild  |
-| members  | array of [guild members](#DOCS_RESOURCES_GUILD/guild-member-object) | set of guild members |
+| members  | array of [member](#DOCS_RESOURCES_GUILD/guild-member-object) objects | set of guild members |
```

Comment:
I think it'd be better to leave the word `guild` in there, personally. The header that it links to is called "Guild Member Object", after all.